### PR TITLE
Rename grafeas gem

### LINKED ISSUE #4009: [grafeas] Rename the "grafeas-client" gem to "grafeas"
The google-cloud Rubygems account now has access to the grafeas gem. We should move the current grafeas-client gem to the grafeas gem (and leave a stub in grafeas-client that just depends on grafeas).

### PR DESCRIPTION
**Pull request by blowmage**
* Rename grafeas-client to grafeas
* Create new grafeas-client gem that depends on grafeas

[closes #4009]

### CODE REVIEW

**Comment by dazuma** [comment]
Sorry, looks like this needs to be resolved against the ruby 2.4 work.

**Comment by blowmage** [comment]
Updated.

File: grafeas/synth.py [hunk 1]
```diff
@@ -211,8 +185,15 @@ def escape_braces(match):
     'https://googleapis.github.io/google-cloud-ruby'
 )
 
+# https://github.com/googleapis/gapic-generator/issues/2393
+s.replace(
+    'grafeas.gemspec',
+    'gem.add_development_dependency "rubocop".*$',
+    'gem.add_development_dependency "google-style", "~> 1.24.0"'
+)
```
Comment (dazuma) [inline grafeas/synth.py]: This can be removed once the `ruby.update_gemspec` line above is fixed.

**Comment by dazuma** [comment]
Also need to update the toplevel Gemfile with the path to the "grafeas" gem directory.

File: grafeas/synth.py (hunk 1 above)
Comment (blowmage) [reply grafeas/synth.py]: Should be fixed.

**Comment by blowmage** [comment]
Updated top-level Gemspec and rebased on master.

File: grafeas/synth.py [hunk 2]
```diff
@@ -42,32 +42,11 @@
...
-        'README.md',
-        'lib/grafeas.rb',
-        'lib/grafeas/v1.rb'
-    ],
-    'gem install grafeas\n',
-    'gem install grafeas-client\n'
-)
 
 # Update gemspec to reflect Ruby 2.4
 ruby.update_gemspec('grafeas-client.gemspec')
```
[the remark this answers is not in the document]
Comment (blowmage) [reply grafeas/synth.py]: Fixed.

**Comment by dazuma** [comment]
Those changes look good. One more thing. Can you run `bundle exec rake kokoro:build` and commit the resulting `grafeas.cfg` kokoro config files?

**Comment by blowmage** [comment]
Kokoro configs built and added.

**Pull request merged**